src/components/ProfileInformation/ProfileInformation.types.ts

```diff
@@ -0,0 +1,11 @@
+import { Profile } from 'decentraland-dapps/dist/modules/profile/types'
+
+export type Props = {
+  isLoggedInProfile: boolean
```

Comment:
We need to have a property that lets us identify one of the possible cases:
1. The user is logged in and is seeing its own profile.
2. The user is logged in and is seeing other user's profile.
3. The user is not logged in and is seeing other user's profile.
This is because we need to show the FriendsButton, the friends and the common friends or none of those depending on the cases.
The boolean can only describe two of those. We might need to replace it with something else, like `loggedInAddress`, from which we can get if the user if logged in (if it's not undefined), if the user the profile we're seeing is the same as the logged in one (by comparing the `profileAddress` and `loggedInAddress`) and if it's a public profile being seen from a non logged in user (if `profileAddres` and `profileAddress` are both undefined).